asset/js/setup.js

```diff
@@ -90,12 +90,6 @@ function setupSiteNav() {
   );
 }
 
-function setupPageNav() {
-  jQuery(window).on('activate.bs.scrollspy', (event, obj) => {
-    document.querySelectorAll(`a[href='${obj.relatedTarget}']`).item(0).scrollIntoView(false);
-  });
-}
```

Comment:
Are you planning any workarounds to restore this feature?

Reply:
With the new layout, the sticky `page-nav` now has the height of the `<content>` instead of the viewport. So `scrollIntoView` would not work.

A different work around would need to be implemented, which is an independently scrollable sticky element. I found a good example here [Scrollable Sticky Element](http://leafo.net/sticky-kit/).

Reply:
With the new layout, the sticky `page-nav` now has the height of the `<content>` instead of the viewport. So `scrollIntoView` would not work.

A different work around would need to be implemented, which is an independently scrollable sticky element.

**References**
[Scrollable Sticky Element](http://leafo.net/sticky-kit/)
[Sticky Sidebar](https://abouolia.github.io/sticky-sidebar/)